lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/show/document_timeline_component.rb

```diff
@@ -34,4 +35,14 @@ def time_html(date_time)
       lang: "en",
     )
   end
+
+  def table_rows(version)
+    if version.field_diffs.present?
+      rows = []
+      version.field_diffs.each do |field|
+        rows.append([{ text: field["field_name"].humanize }, { text: field["previous_value"] }, { text: field["new_value"] }])
+      end
+      rows
```

Comment:
You could probably just do `map` here:

```suggestion
      version.field_diffs.map do |field|
        [
          { text: field["field_name"].humanize }, 
          { text: field["previous_value"] }, 
          { text: field["new_value"] },
        ]
      end
```

(Added some linebreaks for readability too)